Lib/annotationlib.py

```diff
@@ -89,12 +89,10 @@ def evaluate(self, *, globals=None, locals=None, type_params=None, owner=None):
                 return value
         if owner is None:
             owner = self.__owner__
-        if type_params is None and owner is None:
```

Comment:
This check felt unnecessarily strict; many ForwardRefs can be evaluated correctly without worrying about type params.